lib/siwapp_web/templates/page/invoices.html.heex

```diff
@@ -0,0 +1,131 @@
+<nav class="navbar is-fixed-bottom hidden" role="navigation" aria-label="main navigation">
+  <div id="navbarBasicExample" class="navbar-menu">
+    <div class="navbar-end">
+      <div class="navbar-item buttons">
+        <button class="button is-danger">Delete</button>
+        <button class="button is-info">Send Email</button>
+        <button class="button is-info">Set Paid</button> 
+        <button class="button is-info">Download PDF</button>               
+        <button class="button is-info">Duplicate</button>					               
+      </div>
+    </div>
+  </div>
+</nav>
+
+<table class="table is-fullwidth is-striped is-bordered is-hoverable">
+	<thead>
```

Comment:
Indent with 4 spaces. This is my last comment about this. You can see this for yourself